substrate/frame/support/procedural/src/pallet/expand/tt_default_parts.rs

```diff
@@ -198,9 +198,9 @@ pub fn expand_tt_default_parts(def: &mut Def) -> proc_macro2::TokenStream {
 		macro_rules! #default_parts_unique_id_v2 {
 			{
 				$caller:tt
-				frame_support = [{ $($frame_support:ident)::* }]
+				your_tt_return = [{ $my_tt_return:path }]
```

Comment:
I actually had the same issue on the polkadot-sdk umbrella crate and ended up using `::tt *` since path seems to also not be enough.